CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanHazFunny
+{
+    public class Jester
+    {
+        private IJokeOutput _JokeOutput;
+        private IJokeService _JokeService;
+
+        public Jester(IJokeOutput jokeOutput, IJokeService jokeService)
+        {
+            if (jokeOutput is null)
+            {
+                throw new ArgumentNullException(nameof(jokeOutput));
+            }
+            else if (jokeService is null)
+            {
+                throw new ArgumentNullException(nameof(jokeService));
+            }
+            else
+            {
+                _JokeOutput = jokeOutput;
+                _JokeService = jokeService;
+            }
```

Comment:
nit: else blocks are not needed.
```suggestion
            if (jokeOutput is null)
            {
                throw new ArgumentNullException(nameof(jokeOutput));
            }
            if (jokeService is null)
            {
                throw new ArgumentNullException(nameof(jokeService));
            }
            _JokeOutput = jokeOutput;
            _JokeService = jokeService;
```